scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -94,13 +94,9 @@
     </div>
     <div class="form-group">
       <div class="row">
-        <div class="col-xs-3">
-          {{ form.thousand_genomes_frequency.label(class="control-label") }}
-          {{ form.thousand_genomes_frequency(class="form-control") }}
-        </div>
-        <div class="col-xs-3">
-          {{ form.exac_frequency.label(class="control-label") }}
-          {{ form.exac_frequency(class="form-control") }}
+        <div class="col-xs-4">
```

Comment:
You remove two xs-3, and add one xs-4, which skews the layout. Perhaps keep the frequency fields small, and allow some space to add more submit buttons later? We may for instance want one for "custom filters" later?

Reply:
I think it is ok to have them slightly skewed for now. As you say it will be better when we add something else.